### PR TITLE
 Update the Creating and Editing pages guide.

### PR DESCRIPTION
**Pull request by rcaballeromx**
Implement the changes to the blog policy as approved by the Steering Committee

### CODE REVIEW

File: content/about/contribute/creating-and-editing-pages/index.md [hunk 1]
```diff
@@ -41,27 +41,33 @@ is the best fit for your content:
...
   </tr>
 
   <tr>
     <td>Blog Post</td>
     <td>
-      A blog post is a timely article on Istio or products and technologies related to it.
+      A blog post is a timely article on Istio or products and technologies related to it. Typically, posts fall in one of the following four categories:
+      <ul>
+      <li>Posts detailing the author’s experience using and configuring Istio, especially those that articulate a novel experience or perspective.</li>
+      <li>Posts highlighting or announcing Istio features.
```
Comment (geeknoid) [inline content/about/contribute/creating-and-editing-pages/index.md]: Missing a closing LI tag.

Comment (rcaballeromx) [reply content/about/contribute/creating-and-editing-pages/index.md]: Done thanks.

**Comment by rcaballeromx** [comment]
@geeknoid All fixed PTAL.

**Pull request merged**